front/app/containers/UsersShowPage/__snapshots__/UsersShowPage.test.tsx.snap

```diff
@@ -59,6 +59,7 @@ exports[`<UsersShowPage /> renders correctly by default 1`] = `
       <UsersShowPage__UserIdeas>
```

Comment:
Would be happy to delete these

Reply:
For sure, but then I'm going to be requested to add tests instead 😁 No time for that now.